pkg/controller/util/owner.go

```diff
@@ -0,0 +1,21 @@
+package util
+
+import (
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetStorageClusterInitializationOwnerRef returns the owner references instance of the storageClusterInitialization
+func GetStorageClusterInitializationOwnerRef(object *ocsv1.StorageClusterInitialization) metav1.OwnerReference {
```

Comment:
You can use the library from controller-runtime instead of having your own. See [hco's example](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/438dd04/pkg/controller/hyperconverged/hyperconverged_controller.go#L315).

Reply:
yup that's a nice idea.

Reply:
done